### PR TITLE
Add warning about static IP configuration

### PR DESCRIPTION
**Pull request by terminalmage**
Also modified the sphinx configuration so that docs builds don't fail after the
removal of the grains outputter a few weeks ago.

### CODE REVIEW

**Comment by terminalmage** [comment]
Oh, where at?

File: salt/modules/lxc.py [hunk 1]
```diff
@@ -458,6 +458,14 @@ def get_network_profile(name=None):
     <tutorial-lxc-profiles>` for more information on how to use network
     profiles.
 
+    .. warning::
+
+        The ``ipv4``, ``ipv6``, ``gateway``, and ``link`` (bridge) settings in
+        network profiles will only work if the container doesnt redefine the
+        network configuration (for example in
+        ``/etc/sysconfig/network-scripts/ifcfg-<interface_name>`` on
+        RHEL/CentOS, or ``/etc/sysconfig/network`` on Debian/Ubuntu/etc.)
```
Comment (kiorky) [inline salt/modules/lxc.py]: /etc/network/interfaces

File: doc/topics/tutorials/lxc.rst [hunk 2]
```diff
@@ -208,7 +208,23 @@ will directly correspond to a parameter in an LXC configuration file (see ``man
...
+
+    salt myminion lxc.create container1 profile=centos network_profile=centos nic_opts='{eth0: {ipv4: 10.0.0.20/24, gateway: 10.0.0.1}}'
+
+.. warning::
+
+    The ``ipv4``, ``ipv6``, ``gateway``, and ``link`` (bridge) settings in
+    network profiles / nic_opts will only work if the container doesnt redefine
+    the network configuration (for example in
+    ``/etc/sysconfig/network-scripts/ifcfg-<interface_name>`` on RHEL/CentOS,
+    or ``/etc/sysconfig/network`` on Debian/Ubuntu/etc.). Use these with
```
Comment (kiorky) [inline doc/topics/tutorials/lxc.rst]: /etc/network/interfaces

File: doc/ref/output/all/salt.output.grains.rst [hunk 3]
```diff
@@ -1,6 +0,0 @@
-==================
```
Comment (kiorky) [inline doc/ref/output/all/salt.output.grains.rst]: is that one normal ?

**Comment by kiorky** [comment]
i manually merged the commit for cleaner history, (authorship maintained, thx !)

**Pull request closed without merging**